src/components/manageParticipants/atoms/SelectParticipant.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+import { Checkbox } from '@chakra-ui/react';
+
+interface SelectParticipantProps {
+  checked: boolean;
+  onChange: () => void;
+  participantEmail: string;
+}
+
+const SelectParticipant: React.FC<SelectParticipantProps> = ({ checked, onChange, participantEmail }) => {
+  const { user } = useAuth0();
+
+  return <Checkbox isChecked={checked} onChange={onChange} isDisabled={user?.email === participantEmail} />;
```

Comment:
Må disables for eier av møtet. Det er forøvrig mulig å slette seg selv, og det går egentlig greit så man trenger ikke disable din egen checkbox.